docker-compose.yml

```diff
@@ -1,17 +1,20 @@
-version: '2'
+version: '3.7'
 services:
   web:
     build:
       context: .
       dockerfile: web.Dockerfile
-    ports:
-      - "5000:5000"
     env_file: .env
     depends_on:
       - db
     volumes:
       - ./:/app
+    ports:
+      - "5000:5000"
   worker:
+    device_requests:
+      - capabilities:
+        - "gpu"    
```

Comment:
abrichr 9 days ago Member
Can you please confirm that this will still work if the device doesn't have a gpu?

Reply:
It does not, however if you remove lines 15-17 when running on a CPU machine, everything works

Reply:
Please add a comment about this

Reply:
https://github.com/MLDSAI/VisionTracing/pull/2/commits/54a1639ccf2fdaf895551f4c6492237355bb1396